### DISCUSSION TITLE
v0.8.1 - A bit more than a Hotfix

### DISCUSSION THREAD

**Discussion by majora2007** [Kareadita/Kavita#2907] [Announcements]
This is just a bit more than a hotfix, but mainly focused on some polish after more users got their hands on the v0.8 along with the updated Image library. From the community support, v0.8 looks to have gone off really well with only a few users reporting minor data loss. Thanks to everyone joining support and bringing up edge cases that were missed. 

I would also like to remind users that you can upvote and comment (or create your own) feature requests on out [Github Discussions](https://github.com/Kareadita/Kavita/discussions) page. 

# Added
- Added: Added SortOrderLocked field for upcoming ability to set it in the UI, allowing customization of sort order on Series Detail page. (This is not yet in the UI, just in the DB and code)

# Changed
- Changed: Changed how often the 1 version out of date check occurs to be every 4-6 hours instead of 1-2 hours.
- Changed: Allow jumpbar to function on pages as long as there is no custom sort 
- Changed: When refusing OPDS download due to lack of role, use 403 Forbid.
- Changed: Don't show any of the internal information on publication status tooltip when the total items is set to 0 (or unset)
- Changed: Updated Comic Vine library when falling back to taking Series name from directory because no Volume tag exists, don't clean specials from the directory name.

# Fixed
- Fixed: Fixed a bug where some series detail pages have a ton of extra space
- Fixed: Fixed a bug where series folders with nested specials would get a bad LowestSeriesFolder meaning series scans wouldn't do anything important.
- Fixed: Fixed a bug with default ordering failing when a series has a special
- Fixed: Fixed a bug where the very out of date modal wouldn't have the correct number of version out of date.
- Fixed: Fixed a bug where stacked image chapters wouldn't move all the files over to cache
- Fixed: Fixed an oversight where brightness control in manga reader wasn't applying to webtoon reader
- Fixed: Drastically sped up switching between card/list mode on series detail for large series
- Fixed: Fixed Continue -100000 when a series is read on series detail page
- Fixed: When parsing from fallback libraries, use comic parsing only for Comic/Comic Vine library types.
- Fixed: Fixed a bug where weblinks remove would not work well. Now it works more like you would assume.
- Fixed: Fixed a bug where active tab wouldn't default to volumes on comic libraries when there were no issues or specials.
- Fixed: Fixed a bug where progress bar on kavita+ metadata progress wasn't showing red.
- Fixed: Fixed a bug where on Chrome bootstrap sliders were blue and not the primary color.
- Fixed: Fixed a bug where chapters could duplicate due to some regions using ',' as a decimal separator. Now in Kavita, whenever we stringify any numbers, it will always choose English's way as that is how it was designed.
- Fixed: Fixed a case where non-English locales wouldn't parse floats correctly for x.x chapters.
- Fixed: Fixed an exception being thrown on startup for fresh users

# API
- Deprecated series/chapter-metadata API in favor of just including the metadata with all chapter entities.

<hr /><em>This discussion was created from the release <a href='https://github.com/Kareadita/Kavita/releases/tag/v0.8.1'>v0.8.1 - A bit more than a Hotfix</a>.</em>

**Comment by DreamFatigue** [comment]
Hello,

The "How to Update" button on the "New Update Available!" pop-up redirects to a "404 - This page could not be found". I'm having trouble switching to the updated version and would appreciate the guide.

Thanks

**Reply by DieselTech** [reply]
What version are you coming from? The links were updated in the last release of Kavita. Maybe your client is caching an old link, but it should be pointing to here:

https://wiki.kavitareader.com/guides/updating

**Reply by DreamFatigue** [reply]
I'm updating from 0.7.14.0, so that's probably why. Let me follow this guide and let you know. Thanks

**Reply by DreamFatigue** [reply]
Wonderful! Worked like a charm. I appreciate this program and all the work you put in!
